src/App.tsx

```diff
@@ -1,41 +1,63 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
+import { getAllUsers } from './api/api';
+import { User } from './react-app-env';
 
-const App: React.FC = () => (
-  <div className="App">
-    <header className="App__header">
-      <label>
-        Select a user: &nbsp;
+const App: React.FC = () => {
+  const [allUsers, setAllUsers] = useState<User[]>([]);
+  const [currentUser, setCurrentUser] = useState('0');
+  const [selectedPostId, setSelectedPostId] = useState<number>();
```

Comment:
Do you know what type has currentUser and selectedPost?
Will be better if we take types from User and Post interfaces.
Also, if we don't have any value in the state but we know that we will have value - we put in the initial value "null". In this case - we can set 0
```suggestion
  const [currentUserId, setCurrentUserId] = useState<User['id']>(0);
  const [selectedPostId, setSelectedPostId] = useState<Post[id]>(0);
```